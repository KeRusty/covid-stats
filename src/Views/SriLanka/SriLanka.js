import React from 'react';
import { View } from 'react-native';

import Header from '../../Components/Header/Header';

import styles from './SriLanka-styles'

export default function SriLanka(props) {

    return (
        <View style={styles.container}>

            <Header showBack={true} nav={props} />

        </View>
    );
}