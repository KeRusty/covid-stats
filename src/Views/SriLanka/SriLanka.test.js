import React from 'react';
import renderer from 'react-test-renderer';

import SriLanka from './SriLanka';

describe('<SriLanka />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<SriLanka />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});