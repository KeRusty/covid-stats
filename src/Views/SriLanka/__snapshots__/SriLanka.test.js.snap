// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SriLanka /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      }
    }
  >
    <View>
      <RCTSafeAreaView
        emulateUnlessSupported={true}
        style={
          Object {
            "backgroundColor": "#6a59f2",
          }
        }
      >
        <View
          hasTabs={true}
          style={
            Array [
              Object {
                "backgroundColor": "#F8F8F8",
                "borderBottomColor": "#a7a6ab",
                "borderBottomWidth": null,
                "elevation": 0,
                "flexDirection": "row",
                "height": 64,
                "justifyContent": "center",
                "left": 0,
                "paddingLeft": 6,
                "paddingRight": 10,
                "right": 0,
                "shadowColor": null,
                "shadowOffset": null,
                "shadowOpacity": null,
                "shadowRadius": null,
                "top": 0,
              },
              Object {
                "backgroundColor": "#6a59f2",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "alignSelf": "center",
                "flex": 1,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              forwardedRef={[Function]}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": null,
                  "backgroundColor": "transparent",
                  "borderBottomWidth": null,
                  "borderColor": null,
                  "borderLeftWidth": null,
                  "borderRadius": 5,
                  "borderRightWidth": null,
                  "borderTopWidth": null,
                  "borderWidth": undefined,
                  "elevation": 0,
                  "flexDirection": "row",
                  "height": 45,
                  "justifyContent": "space-between",
                  "marginLeft": -3,
                  "opacity": 1,
                  "paddingBottom": 6,
                  "paddingLeft": 4,
                  "paddingRight": 6,
                  "paddingTop": 6,
                  "shadowColor": null,
                  "shadowOffset": null,
                  "shadowOpacity": null,
                  "shadowRadius": null,
                }
              }
            >
              <Text />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flex": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#ffffff",
                  "fontSize": 12,
                  "fontWeight": "bold",
                }
              }
            >
              Sri Lanka Stats
            </Text>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-end",
                "alignSelf": "center",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "flex-end",
              }
            }
          />
        </View>
      </RCTSafeAreaView>
    </View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flex": 1,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "borderBottomWidth": 0.5,
              "borderColor": "#a7a6ab",
              "elevation": 3,
              "flexDirection": "row",
              "height": 50,
              "justifyContent": "space-around",
              "shadowColor": undefined,
              "shadowOffset": undefined,
              "shadowOpacity": undefined,
              "shadowRadius": undefined,
            },
            Object {
              "backgroundColor": "#F8F8F8",
            },
            Object {},
          ]
        }
      >
        <View
          accessibilityRole="tab"
          accessibilityState={
            Object {
              "disabled": false,
              "selected": true,
            }
          }
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flex": 1,
              "opacity": 1,
            }
          }
        >
          <View
            active={true}
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F8F8F8",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "center",
                },
                Object {
                  "backgroundColor": "#6a59f2",
                  "fontWeight": "bold",
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#007aff",
                    "fontFamily": "System",
                    "fontSize": 16,
                    "fontWeight": "600",
                    "marginLeft": 7,
                    "marginRight": 7,
                  },
                  Object {
                    "fontSize": 15,
                  },
                  Object {
                    "color": "#ffffff",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            >
              Local
            </Text>
          </View>
        </View>
        <View
          accessibilityRole="tab"
          accessibilityState={
            Object {
              "disabled": false,
              "selected": false,
            }
          }
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flex": 1,
              "opacity": 1,
            }
          }
        >
          <View
            active={false}
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F8F8F8",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "center",
                },
                Object {
                  "backgroundColor": "#6a59f2",
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#6b6b6b",
                    "fontFamily": "System",
                    "fontSize": 16,
                    "marginLeft": 7,
                    "marginRight": 7,
                  },
                  Object {
                    "fontSize": 15,
                  },
                  Object {
                    "color": "#ffffff",
                    "fontWeight": "bold",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            >
              Global
            </Text>
          </View>
        </View>
        <View
          accessibilityRole="tab"
          accessibilityState={
            Object {
              "disabled": false,
              "selected": false,
            }
          }
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flex": 1,
              "opacity": 1,
            }
          }
        >
          <View
            active={false}
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F8F8F8",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "center",
                },
                Object {
                  "backgroundColor": "#6a59f2",
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#6b6b6b",
                    "fontFamily": "System",
                    "fontSize": 16,
                    "marginLeft": 7,
                    "marginRight": 7,
                  },
                  Object {
                    "fontSize": 15,
                  },
                  Object {
                    "color": "#ffffff",
                    "fontWeight": "bold",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            >
              Hospital Info
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "backgroundColor": "#eae8fd",
              "bottom": 0,
              "height": 4,
              "left": 0,
              "position": "absolute",
              "width": 250,
            }
          }
        />
      </View>
      <RCTScrollView
        alwaysBounceVertical={false}
        automaticallyAdjustContentInsets={false}
        contentOffset={
          Object {
            "x": 0,
          }
        }
        directionalLockEnabled={true}
        horizontal={true}
        keyboardDismissMode="on-drag"
        keyboardShouldPersistTaps="handled"
        onMomentumScrollBegin={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        pagingEnabled={true}
        scrollEnabled={true}
        scrollEventThrottle={16}
        scrollsToTop={false}
        showsHorizontalScrollIndicator={false}
      >
        <View>
          <View
            style={
              Object {
                "width": 750,
              }
            }
          >
            <View
              activeTabStyle={
                Object {
                  "backgroundColor": "#6a59f2",
                  "fontWeight": "bold",
                }
              }
              activeTextStyle={
                Object {
                  "color": "#ffffff",
                }
              }
              heading="Local"
              style={
                Object {
                  "backgroundColor": "#FFF",
                  "flex": 1,
                }
              }
              tabStyle={
                Object {
                  "backgroundColor": "#6a59f2",
                }
              }
              textStyle={
                Object {
                  "color": "#ffffff",
                  "fontWeight": "bold",
                }
              }
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#fff",
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "width": 750,
              }
            }
          >
            <View
              heading="Global"
            />
          </View>
          <View
            style={
              Object {
                "width": 750,
              }
            }
          >
            <View
              heading="Hospital Info"
            />
          </View>
        </View>
      </RCTScrollView>
    </View>
  </View>
</View>
`;
